MetaMask/metamask-mobile
# feat: add account display component

## **Description**



This adds a component that can display the stylized address and account label for the currently selected account.
The component gets used in the `ReceiveRequest` QR code view.

This PR also updates the unit tests and related snapshots for the `ReceiveRequest` view, as the previous unit tests didn't validate the correct snapshot.

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2574
This is part of the https://github.com/MetaMask/MetaMask-planning/issues/2384 epic

The final component depends on some typography fixes from [this PR](https://github.com/MetaMask/metamask-mobile/pull/9962).

## **Manual testing steps**

1. From the home screen, open the bottom drawer
2. Pick `Receive`
3. Observe the Account label and stylized address and copy button under the QR code.

## **Screenshots/Recordings**



### **Before**


<img width="247" alt="Screenshot 2024-06-18 at 13 00 39" src="https://github.com/MetaMask/metamask-mobile/assets/965762/2c477a3d-e1b5-4b39-a283-1fc346c4c4f5">


### **After**

<img width="242" alt="new fonts" src="https://github.com/MetaMask/metamask-mobile/assets/965762/599f0524-badc-436f-bc8f-34c41dbdd79d">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
